Core.DomainServices/SystemUsage/ItSystemUsageOverviewReadModelUpdate.cs

```diff
@@ -57,6 +57,7 @@ public void Apply(ItSystemUsage source, ItSystemUsageOverviewReadModel destinati
             destination.SystemName = source.ItSystem.Name;
             destination.ItSystemDisabled = source.ItSystem.Disabled;
             destination.IsActive = source.IsActive;
+            destination.Note = source.Note;
```

Comment:
`destination.Note = source.Note ?? "";`
This will ensure that using echoSourceValue will not render "null" in the excel output